server/src/main/java/org/elasticsearch/action/admin/cluster/stats/CCSUsage.java

```diff
@@ -133,6 +144,9 @@ public static Result getFailureType(Exception e) {
             if (ExceptionsHelper.unwrapCorruption(e) != null) {
                 return Result.CORRUPTION;
             }
+            if (ExceptionsHelper.unwrap(e, ElasticsearchStatusException.class) != null) {
```

Comment:
This could break in the future if another `ElasticsearchStatusException` gets used for another exception that isn't license based. I would probably also look at the message to at least search for the word "license" and possibly the whole text like I have in some tests: https://github.com/elastic/elasticsearch/pull/118102/files#diff-d4a9ea2a40f283580518f9c52c9bea5fac421a22d92aab3da710a31206c1b413R162